client/client.go

```diff
@@ -246,8 +250,10 @@ func IsTwoFactorError(err error) bool {
 
 // SysInfo holds system information
 type SysInfo struct {
-	Series  string `json:"series,omitempty"`
-	Version string `json:"version,omitempty"`
+	Series    string `json:"series,omitempty"`
+	Version   string `json:"version,omitempty"`
+	OS        string `json:"os-id,omitempty"`
+	OnClassic bool   `json:"on-classic,omitempty"`
```

Comment:
Let's ponder about it conceptually: is it fair to call a snapd-only system as ubuntu 16.04?